test/e2e/framework/pod/resize.go

```diff
@@ -335,10 +335,12 @@ func VerifyPodContainersCgroupValues(ctx context.Context, f *framework.Framework
 				cpuQuota = -1
 			}
 			expectedCPULimitString = strconv.FormatInt(cpuQuota, 10)
+			expectedCPULimitStringRounded := strconv.FormatInt(RoundUp(cpuQuota), 10)
```

Comment:
How about extracting the construction of these expectations to a helper (shared with the pod-level-resources test)? Then we can easily leave a TODO to remove the rounding exception once the runc bug is fixed.

Something like:
```go
func getCPULimitCgroupExpectations(milliCPULimit int64) []string
```

Reply:
done!